documentation/release_notes.markdown

```diff
@@ -4,6 +4,67 @@ title: "Puppet Server: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
+[layout]: https://github.com/puppetlabs/puppet-specifications/blob/2818c90163837ae6a45eb070cf9f6edfb39a1e3f/file_paths.md
+[pup4install]: /puppet/latest/reference/install_linux.html
+[semver]: http://semver.org/
+
+
+
+## Puppet Server 2.1.1
+
+Released June 17, 2015
+
+This is a security and bug fix release in the Puppet Server 2.1 series; no new features have been added since 2.1.0. We recommend that all users upgrade.
+
+### Bug Fixes
+
+#### Support for Loading Facter 3.0+ / Puppet Agent 1.2.0+ With Multiple JRubyPuppet Instances.
+
+Puppet Server uses Facter for a few things. Facter used to be pure Ruby code, but as of puppet-agent 1.2.0 (which includes Facter 3.0.0), it's a native shared library.
+
+Puppet Server 2.1.0 had a problem with loading that shared library if the `jruby-puppet.max-active-instances` setting in `puppetserver.conf` was set higher than 1 (which it almost always is). This bug is now fixed.
+
+* [SERVER-718](https://tickets.puppetlabs.com/browse/SERVER-718)
+
+#### Fix Defaults for puppetserver.conf Directory Settings
+
+The `jruby-puppet` section of puppetserver.conf has five directory settings (`master-conf-dir`, etc.) for setting the [configuration directory][], [code directory][], [cache directory][], run directory, and log directory. When these settings were absent, Puppet Server wasn't properly enforcing the proper default values and could sometimes use the wrong directories, to varying effects.
```

Comment:
Just curious why the configuration, code, and cache directory have []s around them whereas the run and log ones don't.